### DISCUSSION TITLE
ESP32 for BT to MQTT?

### DISCUSSION THREAD

**Discussion by fokcuk** [SBFspot/SBFspot#469] [Q&A]
Hi everyone
Anyone knows of a way to port the connectivity code to ESP32? Want it to connect to the inverter via BT and then send the data via MQTT

**Comment by sbf-** [comment]
Maybe [this](https://github.com/stuartpittaway/nanodesmapvmonitor) can help. SMAspot/SBFspot was originally based on it.
